catalog/task/collect-data/0.3/collect-data.yaml

```diff
@@ -61,17 +75,24 @@ spec:
           mkdir -p $SUBDIR
         fi
 
+        echo -n "${SUBDIR}" > $(results.resultDirectory.path)
+
         cd $SUBDIR
 
+        echo -n "${SUBDIR}/release.json" > $(results.release.path)
```

Comment:
This is wrong. This definitely all needs to be relative to the workspace directory. You cannot work with absolute paths among multiple tasks. If just for the fact that workspace names can be different (config vs. data). But even if we always used the same, we should always rely on `$(workspaces.data.path)` or similar inside tasks.

Reply:
Good call, fixed. It does make me question why we want the `resultsDirectory` result though when we already have `params.subdirectory` and they will be the same

Reply:
Yeah, I don't think we need that one. I think we just need each of the files.

Reply:
Yep.

Reply:
Removed resultDirectory result